docs/vsphere-golden-image.md

```diff
@@ -15,15 +15,23 @@ the patch is not available.*
 ## 2. Create the virtual machine
 
 To start with, create a new virtual machine in vSphere from the selected Windows Server distribution using the ISO image.
-See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-7834894B-DD17-4D59-A9BF-A33D02478521.html)
+See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines.
 
 ### Setup VMware Tools
 
-Install VMware Tools version 11.0.6 or greater. See the [VMware Tools documentation](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-D8892B15-73A5-4FCE-AB7D-56C2C90BD951.html) for more information.
+Install VMware Tools version 11.0.6 or greater. See the [VMware Tools documentation](https://docs.vmware.com/en/VMware-Tools/index.html) for more information.
 
-Enable the VMware Tools configuration file [tools.conf](https://docs.vmware.com/en/VMware-Tools/11.2.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html) within the Windows virtual machine.
+Ensure the virtual machine has a valid IP address in vCenter. Refer to the *IP Addresses* section in the *Summary* tab
+of the vSphere Web Client or run the `ipconfig` command in the Windows VM.
```

Comment:
Re-reading this I see that it is in the incorrect spot. It is best the user checks this after they have configured VMtools.

Reply:
Makes sense, fixed in [9d87266](https://github.com/openshift/windows-machine-config-operator/commit/9d872669e930557d4cb4dc4539fd2299e3fc2603#diff-eff7340e15df41ff3a96507abf744eb5fea3f670fbb381edc624b52cda6b1b79R55)